### PR TITLE
ANDROID-10255 - Adding support for snapshots

### PR DESCRIPTION
**Pull request by gmerinojimenez**
### :goal_net: What's the goal?
Adding support for releasing snapshots

### :construction: How do we do it?
Just adding the action and setting properly the version

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -36,7 +36,11 @@ nexusStaging {
 allprojects {
 
     group = 'com.telefonica.mistica'
-    version = System.getProperty("LIBRARY_VERSION") ?: "undefined"
+    if (System.getProperty("SNAPSHOT_VERSION") != null) {
+        version = System.getProperty("SNAPSHOT_VERSION")+"-SNAPSHOT"
```
Comment (yamal-alm) [inline build.gradle:40]: nice 👍

**Review by dagonco — APPROVED** [review]
That's wonderful @gmerinojimenez! 😇

**Review by yamal-alm — APPROVED** [review]
thanks

**Comment by ieduardogf** [comment]
Esta muy bien el soporte de snapshots, deberíamos de replicarlo tb en mística iOS . 
Nos lo apuntamos

**Pull request merged**